test/integration_test.go

```diff
@@ -148,18 +149,24 @@ func (ts *IntegrationTestSuite) SetupTest() {
 	ts.seq++
 	ts.activities.clearInvoked()
 	ts.taskListName = fmt.Sprintf("tl-%v", ts.seq)
-	ts.worker = worker.New(ts.rpcClient.Interface, domainName, ts.taskListName, worker.Options{
-		DisableStickyExecution: ts.config.IsStickyOff,
```

Comment:
any idea when this would be false?  if ever?  it being not-hard-coded sorta implies it's controllable somehow.

Reply:
bleh, nevermind.  this is maintained in the new code, I just derped